packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -319,6 +320,46 @@ QUnit.module('Chat', moduleConfig, () => {
             assert.strictEqual(getMessageGroups().length, 1);
         });
 
+        QUnit.test('Message Group should rendered in the scrollable content', function(assert) {
+            const newMessage = {
+                author: { id: MOCK_CURRENT_USER_ID },
+                timestamp: NOW,
+                text: 'NEW MESSAGE',
+            };
+
+            this.reinit({ items: [newMessage] });
+
+            const $scrollableContent = this.$element.find(`.${SCROLLABLE_CONTENT_CLASS}`);
+            const $messageGroups = $scrollableContent.find(`.${CHAT_MESSAGE_GROUP_CLASS}`);
+
+
+            assert.strictEqual($messageGroups.length, 1);
+        });
+
+        QUnit.test('Message Group should rendered in the scrollable content after adding 1 new message', function(assert) {
+            const newMessage = {
+                author: { id: MOCK_CURRENT_USER_ID },
+                timestamp: NOW,
+                text: 'NEW MESSAGE',
+            };
+
+            this.instance.option({ items: [newMessage] });
```

Comment:
Let's concider scenario when MessageList already has rendered messages, and will add message the following way:

this.instance.option('items', [...items, newMessage]);

Reply:
[b25a2fe9c8f5b60493f06ce66350f8716c1718b2]